src/pages/organize/[orgId]/journeys/[journeyId]/new.tsx

```diff
@@ -0,0 +1,208 @@
+            }}
+          >
+            <Chip
+              label={intl.formatMessage({
+                id: 'pages.organizeNewJourneyInstance.draft',
+              })}
+              style={{
+                backgroundColor: theme.palette.grey['300'],
+                fontWeight: 'bold',
+              }}
+            />
+          </Box>
+        }
+        title={
+          <EditTextinPlace
+            allowEmpty={true}
+            onChange={async (value) => setTitle(value)}
+            placeholder={intl.formatMessage(
+              { id: 'pages.organizeNewJourneyInstance.title' },
+              { journey: journey.singular_label }
+            )}
+            value={title}
+          />
+        }
+      />
+      <Box p={3}>
+        <Grid container justifyContent="space-between" spacing={2}>
+          <Grid item md={6}>
+            <ZetkinAutoTextArea
+              onChange={(value) => {
+                setNote(value);
+                setEditedNote(true);
+              }}
+              value={note}
+            />
+            <form
+              onSubmit={async (ev) => {
+                ev.preventDefault();
+                mutation.mutate(
+                  { assignees, note, subjects, title },
+                  {
+                    onError: () => {
+                      showSnackbar('error');
+                    },
+                    onSuccess: (newInstance) => {
+                      router.push(
+                        `/organize/${orgId}/journeys/${journeyId}/${newInstance.id}`
+                      );
+                    },
+                  }
+                );
+              }}
+            >
+              <SubmitCancelButtons
+                onCancel={() => {
+                  router.push(`/organize/${orgId}/journeys/${journeyId}`);
+                }}
+                submitDisabled={
+                  !editedNote || mutation.isLoading || mutation.isSuccess
+                }
+                submitText={intl.formatMessage(
+                  {
+                    id: 'pages.organizeNewJourneyInstance.submitLabel',
+                  },
+                  { journey: journey.singular_label }
+                )}
+              />
+            </form>
+          </Grid>
+          <Grid item md={4}>
+            <JourneyInstanceSidebar
+              journeyInstance={{
+                assignees,
+                next_milestone: null,
+                subjects,
+              }}
+              onAddAssignee={(person) => setAssignees([...assignees, person])}
+              onAddSubject={(person) => setSubjects([...subjects, person])}
+              onRemoveAssignee={(person) =>
+                setAssignees(
+                  assignees.filter((assignee) => assignee.id != person.id)
+                )
+              }
+              onRemoveSubject={(person) =>
+                setSubjects(
+                  subjects.filter((subject) => subject.id != person.id)
+                )
```

Comment:
I like this more inline as well

Reply:
Thanks! Yeah the component was really "tall" in my editor only to accommodate having these defined outside, so I tried moving them inline to make it easier to get an overview of the code (i.e. being able to see more of it without scrolling).